### ISSUE TITLE
How to integrate with NextJs 13+

### ISSUE THREAD

**Issue by srosato** [lahmatiy/react-render-tracker#38]
I tried adding the script using `next/script` within `layout.tsx`

```ts
import React from 'react';
import { Providers } from './providers';
import NextTopLoader from 'nextjs-toploader';
import { font } from '@ss/theme/font';
import Script from 'next/script'

export const metadata = {
  title: 'Shadow Shifts - Admin',
};

export default function RootLayout({ children }: { children: React.ReactNode }) {
  return (
    <html lang="en" className={font.className}>
      <Script
        src="https://cdn.jsdelivr.net/npm/react-render-tracker"
      />
      <body>
        <Providers>
          <NextTopLoader />
          {children}
        </Providers>
      </body>
    </html>
  );
}
```

I tried option #0 I get `Waiting for a supported React renderer to be connected...`

![image](https://github.com/lahmatiy/react-render-tracker/assets/650192/9aeb9aee-ace6-4a4b-bad6-5ef8390091ad)

I tried option #1 I get `Connection to rempl publisher – Awaiting...`

![image](https://github.com/lahmatiy/react-render-tracker/assets/650192/430ccec6-c342-4f0f-a41f-150e18ac3df3)

I can see the script loading correctly

![image](https://github.com/lahmatiy/react-render-tracker/assets/650192/8b67464e-2413-4d30-9d00-f5f163e261a2)

![image](https://github.com/lahmatiy/react-render-tracker/assets/650192/539a2ce3-6511-4eeb-9eca-bbb947815371)

Also worth mentioning that I am using React 18

**Comment by aldrinjenson** [comment]
Yeah any updates on this? coming from nextJs 14. This approach doesn't work

**Comment by iAdramelk** [comment]
I was able to make it work with `strategy="beforeInteractive"`:


```
<Script
  src="https://cdn.jsdelivr.net/npm/react-render-tracker"
   data-config="inpage:true"
   strategy="beforeInteractive"
/>
```

But it didn't help that much because it only shows top 20-30 components and does not show anything inside the topmost server component.
